src/pages/staking/index.js

```diff
@@ -1,422 +1,758 @@
+            </p>
+          </Content>
+          <Divider />
+          <Content>
+            <h2 id={tocItems.comparisonOfOptions.id}>
+              {tocItems.comparisonOfOptions.title}
+            </h2>
+            <p>
+              There is no one-size-fits-all solution for staking, and each is
+              unique. Here we'll compare some of the risks, rewards and
+              requirements of the different ways you can stake.
+            </p>
+            <ComparisonGrid>
+              <ColorH3 color="stakingGold">Solo staking</ColorH3>
+              <div
+                style={{
+                  gridArea: "solo-rewards",
+                  borderBottom: "1px solid #3335",
+                }}
+              >
+                <h4>Rewards</h4>
+                <ul>
+                  <li>
+                    Maximum rewards - receive full rewards directly from the
+                    protocol
+                  </li>
+                  <li>
+                    You'll get rewards for batching transactions into a new
+                    block or checking the work of other validators to keep the
+                    chain running securely
+                  </li>
+                  <li>
+                    After The Merge you'll receive unburnt transaction fees for
+                    blocks you propose
+                  </li>
+                </ul>
+              </div>
+              <div
+                style={{
+                  gridArea: "solo-risks",
+                  borderBottom: "1px solid #3335",
+                }}
+              >
+                <h4>Risks</h4>
+                <ul>
+                  <li>Your ETH is at stake</li>
+                  <li>
+                    There are penalties, which cost ETH, for going offline
+                  </li>
+                  <li>
+                    Malicious behavior can result in "slashing" of larger
+                    amounts of ETH and forced ejection from the network
+                  </li>
+                </ul>
+              </div>
+              <div style={{ gridArea: "solo-reqs" }}>
+                <h4>Requirements</h4>
+                <ul>
+                  <li>You must deposit 32 ETH</li>
+                  <li>
+                    Maintain hardware that runs both an Ethereum execution
+                    client and consensus client while connected to the internet
+                  </li>
+                  <li>
+                    The{" "}
+                    <Link to="https://prater.launchpad.ethereum.org">
+                      Staking Launchpad
+                    </Link>{" "}
+                    will walk you through the process and hardware requirements
+                  </li>
+                </ul>
+              </div>
+              <div style={{ gridArea: "solo-cta" }}>
+                <StyledButtonLink
+                  onClick={() => {
+                    trackCustomEvent({
```

Comment:
Let's remove these events - this is just a link, we can track this via pageviews.